extensions/Worker.Extensions.Http.AspNetCore/src/FunctionContext/AspNetExtensionHttpRequestDataFeature.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
+{
+    internal class AspNetExtensionHttpRequestDataFeature : IHttpRequestDataFeature
```

Comment:
Put a placeholder name since I kept blanking on this, but would love suggestions.

Reply:
`AspNetCoreHttpRequestDataFeature` perhaps?